vc/pokeyellow.constants.asm

```diff
@@ -0,0 +1,69 @@
+INCLUDE "constants.asm"
+
+; These are all the asm constants needed to make the blue_vc patch.
+
+vc_const: MACRO
+x = \1
+	PRINTLN "00:{04x:x} \1" ; same format as rgblink's .sym file
+ENDM
+
+; [FPA 031801 Begin1]
+YellowIntroScene14_Index = $e
```

Comment:
This looks like the right time to introduce a file with actual `const_def`/`const` sequences for the Yellow-only Pikachu-related data, even if they're just placeholder constants like some of the ones in move_animation_constants.asm.